src/components/Loader/PeopleTable/PeopleList.tsx

```diff
@@ -0,0 +1,32 @@
+import { Person } from '../../../types';
+import { PersonItem } from './PersonItem';
+
+interface PeopleListProp {
+  people: Person[];
+}
+
+export const PeopleList: React.FC<PeopleListProp> = ({ people }) => {
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => (
+          <PersonItem key={person.slug} person={person} />
+        ))}
+      </tbody>
```

Comment:
Consider adding a conditional rendering to handle the scenario where the 'people' array is empty. You could display a message indicating that there are no people to show, which would enhance the user experience in case the list is empty.